src/lib/project-saver-hoc.jsx

```diff
@@ -41,16 +43,39 @@ const ProjectSaverHOC = function (WrappedComponent) {
                     });
             }
             // TODO: distinguish between creating new, remixing, and saving as a copy
-            if (this.props.isCreating && !prevProps.isCreating) {
-                this.storeProject()
+            if (this.props.isCreatingNew && !prevProps.isCreatingNew) {
+                this.storeProject(null)
                     .then(response => {
                         this.props.onCreatedProject(response.id.toString(), this.props.loadingState);
                     })
                     .catch(err => {
-                        // NOTE: should throw up a notice for user
```

Comment:
No, but the error is being handled in the main error display code in the gui container, there's an issue for it ( https://github.com/LLK/scratch-gui/issues/3324 ), and an identical comment above. Since I was going to replicate similar code for save as a copy and remix, I didn't want to include this comment so many times.